modules/pruning-images.adoc

```diff
@@ -5,111 +5,152 @@
 [id="pruning-images_{context}"]
 = Pruning images
 
-In order to prune images that are no longer required by the system due to age,
-status, or exceed limits, administrators can run the following command:
+Administrators can manually prune images that are no longer required by the system due to age, status, or exceed limits. There are two methods to manually prune images:
 
+* Running image pruning as a `Job` or `CronJob` on the cluster.
+* Running the `oc adm prune images [<options>]` command.
```

Comment:
Do we need `[<options>]` here?  
_Running the `oc adm prune images` command._ and introduce the options later?

Reply:
That makes sense. Thanks!